### PR TITLE
Rawdata server: add a quick startup script for development purpose.

### PR DESCRIPTION
**Pull request by FunkySayu**

### CODE REVIEW

File: third_party/python/riotwatcher/riotwatcher.py [hunk 1]
```diff
@@ -244,6 +244,10 @@ def request_available(self):
 
 
 class RiotWatcher:
+
+    # Used for testing purpose, disabling https make all routes testable.
+    enable_https = True
```
Comment (ArchangelX360) [inline third_party/python/riotwatcher/riotwatcher.py]: Is the comment saying the opposite of what the line does or did I get it wrong?

Comment (FunkySayu) [reply third_party/python/riotwatcher/riotwatcher.py]: Used for testing purpose < i.e. do not modify this for production code
disabling https make all routes testable. < creating an HTTPS server for tests is not a funny thing to do :)
enable_https = True < required by the Riot API

Should I detail a little bit more the comment?

Comment (ArchangelX360) [reply third_party/python/riotwatcher/riotwatcher.py]: "Used for testing purpose < i.e. do not modify this for production code"
This i. e. is way clearer for me than the original sentence.

**Comment by FunkySayu** [comment]
Fixed.

**Review by cassand — APPROVED** [review]
I need this

**Pull request merged**